src/app/components/MessageBanner/index.test.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, screen } from '../react-testing-library-with-providers';
+import MessageBanner from '.';
+import kyrgyzBanner from './fixtures';
+
+const { summaries } = kyrgyzBanner;
+
+describe('MessageBanner', () => {
```

Comment:
I think we need to add a new test to the Topic page tests, to ensure that it renders a message banner when a curation contains an item with visual style = BANNER